ExampleMod/Items/Weapons/ExampleGun.cs

```diff
@@ -12,22 +12,22 @@ public class ExampleGun : ModItem
 		}
 
 		public override void SetDefaults() {
```

Comment:
Please use capitalization, periods and post-commentmarks spacing consistently in your comments, even if the original file doesn't have that.